include/OpenColorIO/OpenColorIO.h

```diff
@@ -883,6 +888,11 @@ class OCIOEXPORT Config
     int getNumViews(const char * display, const char * colorspaceName) const;
     const char * getView(const char * display, const char * colorspaceName, int index) const;
 
```

Comment:
```
/**
 * \brief Compare views in a pair of configs.
 *
 * Will return false if either of the views does not exist. This will return true even
 * if the view is display-defined in one config and a reference to a shared view in the
 * other config (both within the same display), as long as the contents match. The
 * description text (if any) is ignored, since it is not a functional difference.
 */
```